javascript/javascript1/week2/event_application.js

```diff
@@ -0,0 +1,11 @@
+function getEventWeekday(futureEventdayNumber){
+    const today= new Date();
+    const daysOfWeek=['MOnday','Tuesday','Wednsday','Thursday','Friday','Saturday','Sunday'];
+    const dayOfEvent=((today.getDay() + futureEventdayNumber)%7)-1; //since array index starts from 0 
+    console.log(`Today is : ${today}\n \nThe number for the day is: ${today.getDay()}`);
+    console.log(`The number for the event day in future is: ${futureEventdayNumber}`);
+    console.log(`Therefore:the event will be held on : ${daysOfWeek[dayOfEvent]} `);
```

Comment:
The function was supposed to `return` the week day, see the [homework example](https://github.com/HackYourFuture-CPH/JavaScript/blob/master/javascript1/week2/homework.md#event-application):
```js
// With todays weekday a tuesday
console.log(getEventWeekday(9)); // Logs out "Thursday"
```

Your function implementation is printing the result directly to the console instead of returning it.

Imagine the customer who ordered the function wanting to use it in their own code `showEventDayOnWebsite`:
```js
let eventDay = getEventWeekday(9)

showEventDayOnWebsite(eventDay)
```

Because you don't return anything they can't actually use your function (`eventDay` would contain `undefined` in the above example using your implementation of `getEventWeekday`).


The `console.log` lines should be considered temporary (only present while you develop/debug the function), and removed in the final version. You will experience less need for these `console.log` as your skill grows.

Reply:
thank you so much Jacob , yes i will try to use return rather than using console.